### PR TITLE
Added some necessary Youtube playlist of Bengali Language

### PR DESCRIPTION
**Pull request by noobSrijon**
## What does this PR do?
Add resource(s)

### Description
Added some necessary resources for newbie programmers of Bengali language. Even I added some playlists from where I learnt how to code.
### Why is this valuable (or not)?
This PR contains some necessary Youtube Playlists in Bengali Language which may helps a newbie programmer to build up his/her skills

### How do we know it's really free?
Added resources are from Youtube which is completely free

### For book lists, is it a book? For course lists, is it a course? etc.
It is some Youtube Playlist

## Checklist:
- [ x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [ x] Search for duplicates.
- [ x] Include author(s) and platform where appropriate.
- [ x] Put lists in alphabetical order, correct spacing.
- [ x] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by davorpa** [comment]
TIP: use `[x]` to fill tasklisk steps

File: books/free-programming-books-bl.md [hunk 1]
```diff
@@ -104,6 +104,7 @@
 * [JavaScript Behind The Scene Bangla Tutorial](https://www.youtube.com/playlist?list=PL_XxuZqN0xVDPaOrWvTIuhb5GRoJVWiE2) - Stack Learner
 * [JavaScript Full Playlist With Order](https://www.youtube.com/playlist?list=PL_XxuZqN0xVAJTV_1ZXwB1XIiFkK0ddZA) - Stack Learner
 * [Make Fun of JavaScript Array](https://www.youtube.com/playlist?list=PL_XxuZqN0xVDr08QgQHljCecWtA4jBLnS) - Stack Learner
+* [JavaScript Bangla Tutorial](https://www.youtube.com/playlist?list=PLHiZ4m8vCp9OkrURufHpGUUTBjJhO9Ghy) - Learn with Sumit Bangladesh
```
Comment (davorpa) [inline books/free-programming-books-bl.md:107]: Please, ensure fullfill author attributions @learnwithsumit

ℹ️ From his LinkedIn profile: `Sumit Saha`

⚠️ Needs real title: `JavaScript Bangla Tutorial | JS Bangla Tutorial Series for Beginners`

**Review by davorpa — CHANGES_REQUESTED** [review]
ℹ️ Apply CONTRIBUTE guideline notes on the other added resources.

⚠️ Check CI linter logs to ensure list alphabetization

**Comment by eshellman** [comment]
added to books list in another PR

**Comment by davorpa** [comment]
Duplicate of #7002

File: books/free-programming-books-bl.md [hunk 2]
```diff
@@ -151,6 +152,8 @@
 
 * [Python All You Need To Know](https://www.youtube.com/playlist?list=PL_XxuZqN0xVCH8fRfZ8sUuKEQQYIFMjtJ) - Stack Learner
 * [Python Bangla Tutorials for Beginners](https://www.youtube.com/playlist?list=PLgH5QX0i9K3rz5XqMsTk41_j15_6682BN) - Anisul Islam
+* [Bangla Python Tutorial for Beginners](https://www.youtube.com/playlist?list=PLlBKlxyCgmsCYJLq9qc5QzaU-oBFJN79B) - StartBit
```
[the remark this answers is not in the document]
Comment (davorpa) [reply books/free-programming-books-bl.md:155]: Resolved by commit 922e371ff8322268fb50635090326a852d9a499d made at PR #7002

**Pull request closed without merging**